src/domain/Defaults/Defaults/Defaults.tsx

```diff
@@ -81,9 +53,8 @@ const useTranslateFunction = () => {
     return i18nTranslationFunction
   }
 
-  return (i18nKey: keyof typeof i18nDefaults | Array<keyof typeof i18nDefaults>, translationFunctionOptions: {[k: string]: any} = {}) => {
-    const key = isArray(i18nKey) ? i18nKey[0] : i18nKey
-    let translation = i18nDefaults[key]
+  return (i18nKey: string | string[], translationFunctionOptions: {defaultValue: string, [k: string]: any}) => {
+    let translation = translationFunctionOptions.defaultValue
```

Comment:
You moved keys to `uiComponents: ` in `common.yml `file. So now how does this method identify location to `common.uiComponents`. in common.yml in accent ?

Reply:
You pass in a scoped `t` function with the `common.uiComponents` scope as a default.